src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,475 @@
+
+Bug fixes to the underlying metric computation can result in different scores. This affects
+relatively few sites but can have sizable impact in certain situations. Overall, about 8% of sites
+experienced a score improvement due to metric implementation changes and about 4% of sites saw a score
+decrease due to metric implementation changes. Approximately 88% of sites were unaffected by these fixes. 
+
+Individual score curve changes also impacted the overall score shifts although very slightly. We
+periodically ensure that the score curve aligns with the observed metrics in the [HTTPArchive
+dataset](http://httparchive.org/). Excluding sites affected by major implementation changes, minor
+adjustments to the score curve for individual metrics improved the scores of about 3% of sites and
+decreased the scores of about 4% of sites. Approximately 93% of sites were unaffected by this change.
+
+### Scoring calculator {: #calculator }
+
+We've published a [scoring calculator](https://googlechrome.github.io/lighthouse/scorecalc/) to help
+you explore performance scoring. The calculator also gives you a comparison between Lighthouse version 5 and
+6 scores. When you run an audit with Lighthouse 6.0, the report comes with a link to the calculator
+with your results populated.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="lighthouse-scorecalc.png" alt="Lighthouse Score Calculator." width="600">
+  <figcaption class="w-figcaption">Huge thanks to <a href="https://twitter.com/anatudor">Ana Tudor</a> for the gauge upgrade!</figcaption>
+</figure>
+
+## New audits {: #new-audits }
+
+### Unused JavaScript {: #unused-javascript }
+
+We are leveraging [DevTools code
+coverage](https://developers.google.com/web/tools/chrome-devtools/coverage) in a new audit: [Unused
+JavaScript](https://web.dev/remove-unused-code/).
+
+This audit isn't _entirely_ new: it was [added in
+mid-2017](https://github.com/GoogleChrome/lighthouse/issues/1852#issuecomment-306900595), but
+because of the performance overhead it was disabled by default to keep the Lighthouse as fast as
+possible. Collecting this coverage data is far more efficient now, so we feel comfortable enabling
+it by default.
+
+### Accessibility audits {: #a11y }
+
+Lighthouse uses the wonderful [axe-core](https://github.com/dequelabs/axe-core) library to power the
+accessibility category. In Lighthouse 6.0, we've added the following audits:
+
+-  [aria-hidden-body](https://web.dev/aria-hidden-body/)
+-  [aria-hidden-focus](https://web.dev/aria-hidden-focus/)
+-  [aria-input-field-name](https://web.dev/aria-input-field-name/)
+-  [aria-toggle-field-name](https://web.dev/aria-toggle-field-name/)
+-  [form-field-multiple-labels](https://web.dev/form-field-multiple-labels/)
+-  [heading-order](https://web.dev/heading-order/)
+-  [duplicate-id-active](https://web.dev/duplicate-id-active/)
+-  [duplicate-id-aria](https://web.dev/duplicate-id-aria/)
+
```

Comment:
I'm bringing this up late, but could we use the full names of the audits in sentence case instead of the url shorthands?